input/templates/my-account-edit-address.json

```diff
@@ -0,0 +1,18 @@
+{
+	"header" : { "$ref" : "common/header.json" },
+	"footer" : { "$ref" : "common/footer.json" },
+	"content" : {
+		"contentTitle" : "Address Book",
+		"defaultAddressesTitle" : "Default Addresses",
+		"shippingAddress" : {
+			"title" : "Shipping Address",
```

Comment:
An address is neither "shipping" nor "billing" when you define it in your address book. It can be pinned as the default shipping and/or billing address, but you see that by the checkboxes, not by the title. Therefore I would remove this value and the corresponding usage in the template.